sql/core/src/test/scala/org/apache/spark/sql/CollationSuite.scala

```diff
@@ -2175,4 +2175,9 @@ class CollationSuite extends DatasourceV2SQLBase with AdaptiveSparkPlanHelper {
       sql(s"CREATE TABLE t (c STRING COLLATE system.builtin.UTF8_LCASE)")
     }
   }
+
+  test("dsadasda") {
```

Comment:
nice name for the test. How is it related to collation?

Reply:
This was accidental push, reverted.